db-scripts/src/main/resources/cgds.sql

```diff
@@ -340,34 +340,46 @@ CREATE TABLE `sample_profile` (
 
 -- --------------------------------------------------------
 CREATE TABLE `structural_variant` (
-  `SAMPLE_ID` int(11) NOT NULL,
+  `GENETIC_PROFILE_ID` int(11) NOT NULL,
   `INTERNAL_ID` int(11) NOT NULL auto_increment,
-  `BREAKPOINT_TYPE` varchar(25),
-  `ANNOTATION` varchar(255),
-  `COMMENTS` varchar(2048),
-  `CONFIDENCE_CLASS` varchar(25),
-  `CONNECTION_TYPE` varchar(25),
-  `EVENT_INFO` varchar(255),
-  `MAPQ` int(11),
-  `NORMAL_READ_COUNT` int(11),
-  `NORMAL_VARIANT_COUNT` int(11),
-  `PAIRED_END_READ_SUPPORT` varchar(255),
-  `SITE1_CHROM` varchar(25),
-  `SITE1_DESC` varchar(255),
-  `SITE1_ENTREZ_GENE_ID` int(11),
-  `SITE1_POS` int(11),
-  `SITE2_CHROM` varchar(25),
-  `SITE2_DESC` varchar(255),
+  `SAMPLE_ID` int(11) NOT NULL,
+  `SITE1_ENTREZ_GENE_ID` int(11) NOT NULL,
+  `SITE1_ENSEMBL_TRANSCRIPT_ID` varchar(25),
+  `SITE1_EXON` int(4),
+  `SITE1_CHROMOSOME` varchar(25),
```

Comment:
why 25? Make it long enough for `chr22`, so 5.

Reply:
Done